specification/hybridcompute/resource-manager/readme.md

```diff
@@ -0,0 +1,177 @@
+
+``` yaml $(tag) == 'package-2019-03'
+input-file:
+- Microsoft.HybridCompute/preview/2019-03-18/HybridCompute.json
+```
+
+### Tag: package-2019-08
+
+These settings apply only when `--tag=package-2019-08` is specified on the command line.
+
+``` yaml $(tag) == 'package-2019-08'
+input-file:
+- Microsoft.HybridCompute/preview/2019-08-02/HybridCompute.json
+```
+
+---
+# Code Generation
+
+
+---
+## C# 
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+``` yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  namespace: Microsoft.Azure.Management.HybridCompute
+  output-folder: $(csharp-sdks-folder)/HybridCompute/Microsoft.Azure.Management.HybridCompute/src/Generated
+  clear-output-folder: true
+directive:
+  - remove-operation: 
+    - Machines_Reconnect
+    - Machines_CreateOrUpdate
+    - Machines_Update
+    - Machines_Delete
+```
+
+## PowerShell
+
+``` yaml $(powershell)
+powershell:
+  namespace: Microsoft.Azure.Management.HybridCompute
+  output-folder: generated
+  clear-output-folder: true
+directive:
+  - remove-operation: 
+    - Machines_Reconnect
+    - Machines_CreateOrUpdate
+    - Machines_Update
+    - Machines_Delete
+```
+
+
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+python-mode: create
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.hybridcompute
+  package-name: azure-mgmt-hybridcompute
+  clear-output-folder: true
+```
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-hybridcompute/azure/mgmt/hybridcompute
+```
+``` yaml $(python) && $(python-mode) == 'create'
+python:
+  basic-setup-py: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-hybridcompute
+```
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  clear-output-folder: true
+  namespace: hybridcompute
+```
+
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2019-03
+```
+
+### Tag: package-2019-03 and go
+
+These settings apply only when `--tag=package-2019-03 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2019-03' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/hybridcompute/mgmt/2019-03-18/hybridcompute
```

Comment:
Please change this output-folder to
`$(go-sdk-folder)/services/preview/hybridcompute/mgmt/2019-03-18-preview/hybridcompute`

Reply:
updated